targets/unity-v2/source/Client/Authentication/Strategies/EmailPasswordAuthStrategy.cs

```diff
@@ -26,8 +26,42 @@ public void Authenticate(PlayFabAuthService authService, Action<LoginResult> res
             {
                 Email = authService.Email,
                 Password = authService.Password,
-                InfoRequestParameters = authService.InfoRequestParams
-            }, resultCallback, errorCallback);
+                InfoRequestParameters = authService.InfoRequestParams,
+                TitleId = PlayFabSettings.TitleId
+            }, resultCallback, error =>
+            {
+                // If user not registered, create one.
+                // Do it because LoginWithEmailAddressRequest does not contains "CreateAccount" option.
```

Comment:
nitpick, you can probably combine these two comments to a single gramatically structured line.